src/app/slash_plugins/gamble.plugin.ts

```diff
@@ -0,0 +1,98 @@
+import { MessageEmbed } from 'discord.js';
+import { ISlashCommand } from '../../common/slash';
+import { IContainer } from '../../common/types';
+
+const MIN_BET: number = 10;
+
+const plugin: ISlashCommand = {
+  name: 'gamble',
+  commandName: 'gamble',
+  description: 'Bet your Tacos on a coin flip',
+  options: [
+    {
+      name: 'amount',
+      description: 'The amount of Tacos to gamble',
+      type: 'STRING',
+      required: true,
+    },
+  ],
+  execute: async ({ interaction, container }) => {
```

Comment:
Small nit, other plugins use the `async execute(…)` style.

Reply:
oh ok yeah i want to stick to the same pattern. just didnt realize you could do that